level-1/두-개-뽑아서-더하기.py

```diff
@@ -0,0 +1,9 @@
+def solution(numbers):
+    answer = []
+    for i in range(len(numbers)):
+        for j in range(i+1, len(numbers)):
+            if numbers[i]+numbers[j] not in answer:
+                answer.append(numbers[i] + numbers[j])
```

Comment:
I think there's some room to improve time complexity. Analyze time complexity of current algorithm and try to find out with a better solution.

Reply:
Current algorithm has time complexity of O(n^3).

Reply:
The new algorithm has the time complexity of O(n^2).